homeplug_exporter.go

```diff
@@ -396,9 +396,20 @@ func read_homeplug(iface *net.Interface, conn *raw.Conn, ch chan<- HomeplugFrame
 			level.Error(logger).Log("msg", "Failed to unmarshal homeplug frame", "err", err)
 			continue
 		}
-
 		level.Debug(logger).Log("msg", fmt.Sprintf("[%v] %+v", addr, h))
-		ch <- h
+
+		if h.MMEType != nwInfoCnf {
+			level.Error(logger).Log("msg", fmt.Sprintf("Got unhandled MME type: %v", h.MMEType))
```

Comment:
I missed this on the first pass. Generally, structured logging discourages the use `fmt.Sprintf` to embed values in log messages. They should instead be passed as discrete fields, as this allows for easier log parsing by downstream tools such as Loki.
```suggestion
			level.Error(logger).Log("msg", "Got unhandled MME type", "mme_type", h.MMEType)
```